src/components/reusuables/cards/FeatureWorkCard.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+
+const FeatureWorkCard = (props) => {
+  return (
+    <div className="carousel-card">
+      <div className="card-img">
+        <StaticImage src="../../../assets/images/kulhiImage.jpg" />
```

Comment:
Can this be passed as a prop? Import it into the home.js and passed. May need GatsbyImage instead of StaticImage.

Reply:
Feature Work card has been rebuilt to use a dynamic image, also rebuilt the featured work section since we removed the carousels that I installed.

Reply:
Side note: I had to replace the GatsbyImage with a bootstrap image to get it to work.